specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/library.json

```diff
@@ -0,0 +1,479 @@
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Library"
+        ],
+        "operationId": "Library_Get",
+        "description": "Get Library",
+        "x-ms-examples": {
+          "Get Library": {
+            "$ref": "./examples/Library_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LibraryNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/LibraryResource"
+            }
+          },
+          "304": {
+            "description": "Not modified."
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LibraryPayload": {
+      "description": "Library content",
+      "type": "object",
+      "properties": {
+        "content": {
+          "type": "string",
+          "description": "Stream of library content"
+        }
+      }
+    },
+    "OperationResult": {
+      "description": "Operation status for the operation",
+      "type": "object",
+      "properties": {
+        "status": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Operation status"
+        },
+        "error": {
+          "x-ms-client-flatten": true,
+          "description": "Error data",
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      }
+    },
+    "SubResource": {
+      "description": "Azure Synapse nested resource, which belongs to a workspace.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/AzureEntityResource"
+        }
+      ]
+    },
+    "LibraryListResponse": {
+      "description": "A list of Library resources.",
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Library.",
+          "items": {
+            "$ref": "#/definitions/LibraryResource"
+          }
+        },
+        "nextLink": {
+          "description": "The link to the next page of results, if any remaining results exist.",
+          "type": "string"
+        }
+      }
+    },
+    "LibraryResource": {
+      "description": "Library response details",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SubResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/Library",
```

Comment:
>Library [](start = 33, length = 7)

Can we use another name? (Note this doesn't require any backend API change. It only influence the SDK class name). I got the following error when generate track 2 C# SDK:
Error AZC0012 Single word class names are too generic and have high chance of collision with BCL types or types from other libraries